ledger-tool/src/main.rs

```diff
@@ -1689,6 +1699,32 @@ fn main() {
                 .global(true)
                 .help("Use DIR for separate incremental snapshot location"),
         )
+        .arg(
+            Arg::with_name("replaying_backend")
+                .long("replaying-backend")
+                .value_name("BACKEND")
+                .takes_value(true)
+                .possible_values(ReplayingBackend::cli_names())
+                .default_value(ReplayingBackend::default().into())
```

Comment:
Not super familiar with how clap validation works, or if this is a product of us using strum.

I tested this and passing `--replaying-backend` (note I didn't specify the variant) works fine and results in the default.

That seemed a bit odd to me.

Reply:
ugh, so much thanks for catching this: d1d317ef3c15b655e9f26482c44050f0632bdd41 + 14c0f88f8604dab008cf7f447baf86299ba6b6eb